bundles/org.openhab.binding.openthermgateway/README.md

```diff
@@ -32,36 +32,40 @@ The configuration settings for the thing are Hostname/IP address and Port, which
-| dhw_mode             | Switch    | Domestic hot water active                                | yes       |
-| flame                | Switch    | Burner active                                            | yes       |
-| modulevel            | Number    | Relative modulation level                                | yes       |
-| maxrelmdulevel       | Number    | Maximum relative modulation level                        | yes       |
-| fault                | Switch    | Fault indication                                         | yes       |
-| servicerequest       | Switch    | Service required                                         | yes       |
-| lockout-reset        | Switch    | Lockout-reset enabled                                    | yes       |
-| lowwaterpress        | Switch    | Low water pressure fault                                 | yes       |
-| gasflamefault        | Switch    | Gas or flame fault                                       | yes       |
-| airpressfault        | Switch    | Air pressure fault                                       | yes       |
-| waterovtemp          | Switch    | Water over-temperature fault                             | yes       |
-| oemfaultcode         | Switch    | OEM fault code                                           | yes       |
-| sendcommand          | Text      | Channel to send commands to the OpenTherm Gateway device | no        |
+| Channel Type ID          | Item Type | Description                                              | Read Only |
+|--------------------------|-----------|----------------------------------------------------------|-----------|
+| roomtemp                 | Number    | Current sensed room temperature                          | yes       |
+| roomsetpoint             | Number    | Current room temperature setpoint                        | yes       |
+| temperaturetemporary     | Number    | Temporary override room temperature setpoint             | no        |
+| temperatureconstant      | Number    | Constant override room temperature setpoint              | no        |
+| controlsetpoint          | Number    | Central heating water setpoint set at boiler             | yes       |
+| controlsetpointrequested | Number    | Central heating water setpoint requested by thermostat   | yes       |
+| controlsetpointoverride  | Number    | Central heating water setpoint configured at gateway     | yes       |
+| dhwtemp                  | Number    | Domestic hot water temperature                           | yes       |
+| tdhwset                  | Number    | Domestic hot water temperature setpoint                  | yes       |
+| overridedhwsetpoint      | Number    | Domestic hot water temperature setpoint override         | no        |
+| flowtemp                 | Number    | Boiler water temperature                                 | yes       |
+| returntemp               | Number    | Return water temperature                                 | yes       |
```

Comment:
All of these temperature channels should have their item type listed as `Number:Temperature`.

Reply:
Again, I'd followed the pattern of the existing file, but I've corrected all of these and added a couple of missing UOMs in state updates alongside this.